### ISSUE TITLE
Cantus Siglum connect to RISM

### ISSUE THREAD

**Issue by Yueqiao12Zhang** [DDMAL/linkedmusic-datalake#109]
Research about how to track from a Siglum code into a RISM id, and therefore reconcile it to get a link to the RISM database page.

**Comment by dchiller** [comment]
We actually already have this...

**Comment by dchiller** [comment]
@Yueqiao12Zhang In the version of CantusDB that is currently on our staging site (staging.cantusdatabase.org) we have changed how we refer to sources, so you should actually use the csv api from there for now (I can let you know when it switches to production).

**Comment by Yueqiao12Zhang** [comment]
What I'm doing is make an API call to the cantus db csv export using the source ids, and retrieve the CSV. In those CSVs, there are many siglum codes that we can search in RISM.

**Comment by dchiller** [comment]
> What I'm doing is make an API call to the cantus db csv export using the source ids, and retrieve the CSV. In those CSVs, there are many siglum codes that we can search in RISM.

Yes, but I'm saying please don't do this. We have just spent a lot of time reconciling CantusDB sigla against RISM, so it doesn't make sense for you to do this again.

**Comment by Yueqiao12Zhang** [comment]
Ok, then I will just leave the RISM sigla codes as they are.

**Comment by dchiller** [comment]
> Ok, then I will just leave the RISM sigla codes as they are.

For now, yes. I assume that eventually you will want the RISM ID for the source?

**Comment by Yueqiao12Zhang** [comment]
I need the RISM link just like other databases I worked with.

**Comment by Yueqiao12Zhang** [comment]
If the RISM ID is equivalent to the webpage URL of the same RISM chants' source, then I mean the same thing.

**Comment by dchiller** [comment]
It's just a URI for a source.
